_episodes/13-scheduler.md

```diff
@@ -251,9 +251,10 @@ minutes.
 
 ```
 #!/bin/bash
+{{ site.sched.comment }} {{ site.sched.flag.name }} long_job
 {{ site.sched.comment }} {{ site.sched.flag.time }} 00:00:30
 
-echo -n "This script is running on "
+echo -n "This script is running on ..."
```

Comment:
Using `echo -n`, the hostname will print immediately after the `...`
Please add a space after the final `.` so we get `This script is running on ... landing_pad` instead of `This script is running on ...landing_pad`. Thanks!